etas/oef/data/europe_parameters.json

```diff
@@ -0,0 +1,88 @@
+{
+    "name": "europe_ESHMregion_bg_alpha_bpos2",
+    "id": "europe_cut_region_bg_alpha_bpos2",
+    "fn_catalog": false,
```

Comment:
This is very picky, but:
If you think from a "data type" perspective, eg. in python type hinting. This would be `fn_catalog: str | bool` which does not make sense, since it will never be `true`, only `false` or a string. So just use `str | None` (meaning it's optional), and use `if fn_catalog is None` on the Python side.